packages/create-next-app/templates/index.ts

```diff
@@ -118,7 +117,7 @@ export const installTemplate = async ({
   }
 
   if (srcDir) {
-    await makeDir(path.join(root, 'src'))
+    await fs.mkdir(path.join(root, 'src'))
```

Comment:
Do we need to add `recursive: true` here too?

Reply:
Not sure `recursive: true` is necessary here, given that the root should already exist and only a one-level src folder will be created.

Nonetheless, including it won't cause any harm. Addressed in [`786162b` (#58030)](https://github.com/vercel/next.js/pull/58030/commits/786162b26394a37fcf1a2d63c3cedc99a5efb911).